cmd/crank/beta/trace/internal/printer/default.go

```diff
@@ -253,10 +256,19 @@ func getResourceStatus(r *resource.Resource, name string, wide bool) fmt.Stringe
 		status = fmt.Sprintf("%s: %s", status, m)
 	}
 
+	ttr := "-"
+	var readyAt time.Time
+	if readyCond.Status == corev1.ConditionTrue {
+		createdAt := r.Unstructured.GetCreationTimestamp()
+		readyAt = readyCond.LastTransitionTime.Time
```

Comment:
Isn't this brittle? What if the condition flaps? TTR would keep increasing? Maybe this would still be valuable 🤔 I now realized we didn't add AGE as a column 😅

Reply:
> Isn't this brittle?

Any better way to show this? Or doesn't it make sense to have such a column at all?

> What if the condition flaps? TTR would keep increasing? 

If you check the example in the description, this is already happening for composites but MRs seems fine. That needs further investigation I believe. 

But how would we define TTR if resource become `Ready` but then `NotReady` and again back to `Ready`, time until initial Readyness or the latest 🤔

Reply:
Despite all the questions and uncertainties, I can imagining this information could be helpful to identify why a claim/composite takes too much time. May be there is a better way to represent this with a dedicated subcommand in another format.